src/com/google/foam/demos/tabata/index.html

```diff
@@ -12,8 +13,8 @@
       var d = Tabata.create();
       foam.u2.DetailView.create({ data: d, showActions: true }).write();
 //      TabataBarCView.create({ data: d }).write();
-      TabataCView.create({ data: d }).write();
```

Comment:
Do we need to modify this demo?

Reply:
Just the TabataState.js should be required. I'll undo the other changes.